.travis_suppressions

```diff
@@ -2,12 +2,10 @@ unusedPrivateFunction:test/testbufferoverrun.cpp
 unusedPrivateFunction:test/testcmdlineparser.cpp
 redundantNextPrevious:test/testtoken.cpp
 simplePatternError:test/testtoken.cpp
-noValidConfiguration
```

Comment:
This should never happen since we specify a fixed configuration. If it does it means we need to extend it or add a `__CPPCHECK__` case.